WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -26,16 +28,20 @@ final class StoreStatsAndTopPerformersViewController: TabbedViewController {
         return periodVCs[selection]
     }
 
+    var timeRangeCount: Int {
```

Comment:
You're right. I overcomplicated that because I misunderstood how Combine works. Updated in https://github.com/woocommerce/woocommerce-ios/pull/11977/commits/3594b95927e16fbd52889283e6c56665bcf72e88